src/adv.py

```diff
@@ -67,33 +67,55 @@
 #
 # If the user enters "q", quit the game.
 
-user_input = None
+choice = None
+moved = True
+
+while choice != 'q':
+    if (choice in ['n', 's', 'w', 'e', None] and moved):
+        print("===============\n")
+        print(f"{player.name} enters \"{player.room.name}\"")
+        print(f"{player.room.desc}\n")
 
-while user_input is not 'q':
-    print("===============\n")
-    print(f"{player.name} enters \"{player.room.name}\"")
-    print(f"{player.room.desc}\n")
     player.room.print_items()
-    user_input = input(
-        "\nMove (n, s, w, e) or Take Action (get/take/drop/inventory): ")
 
-# Movement Logic
-    if len(user_input.split(' ')) == 1:
-        if (user_input == 'n' or user_input == 'N') and hasattr(player.room, 'n_to'):
+# Player input
+    choice = input(
+        "\nCheck Inventory (i/inv), Move (n, s, w, e) or Take Action (get/take/drop): ")
+    choice_arr = choice.split(' ')
+    choice_len = len(choice_arr)
+# Movement and Player Inventory Logic (expects 1 argument)
+    if choice_len == 1:
+        if (choice in ['n', 'N']) and hasattr(player.room, 'n_to'):
+            moved = True
             player.room = player.room.n_to
-        elif (user_input == 's' or user_input == 'S') and hasattr(player.room, 's_to'):
+        elif (choice in ['s', 'S']) and hasattr(player.room, 's_to'):
+            moved = True
             player.room = player.room.s_to
-        elif (user_input == 'e' or user_input == 'E') and hasattr(player.room, 'e_to'):
+        elif (choice in ['e', 'E']) and hasattr(player.room, 'e_to'):
+            moved = True
             player.room = player.room.e_to
-        elif (user_input == 'w' or user_input == "W") and hasattr(player.room, 'w_to'):
+        elif (choice in ['w', 'W']) and hasattr(player.room, 'w_to'):
+            moved = True
             player.room = player.room.w_to
-        elif user_input == 'i' or user_input == 'inv' or user_input == 'inventory':
+        elif choice == 'i' or choice == 'inv' or choice == 'inventory':
```

Comment:
you can use the same logic here
elif choice in ['i', 'inv', 'inventory']

think of ways of handling typing mistakes and uppercase/lowercase mixes